module1/index.md

```diff
@@ -2,9 +2,72 @@
+
+## Creating hello world app in Next.js
+
+```bash
+npm install -g pnpm
+```
+
+Create an app
 
-Vercel is a PaaS company built on top of AWS.
+``` bash
+pnpx create-next-app@latest --use-pnpm
+```
+The --use-pnpm flag ensures that pnpm is used as the package manager for the project.
+
+Choose a template (optional): After running the command above, you'll be prompted to choose a template for your Next.js app. You can select from options like a default template, a TypeScript template, or an example template. Choose the one that suits your project requirements, or simply press Enter to select the default template.
+
+Navigate to the app directory: Once the project is created, navigate to the app's directory using the following command:
+
+```bash
+cd your-app-name
+```
+
+Start the development server: Now, you can start the development server and run your Next.js app. Use the following command:
+
+```bash
+pnpm dev
+```
+
+This command will start the development server and provide you with a local URL (e.g., http://localhost:3000) where you can view your app.
+
+That's it! You've successfully created a new Next.js app using pnpm as the package manager. You can start building your app by modifying the files in the pages directory and exploring the Next.js documentation for more information on how to work with Next.js features and components.
+
+### Deploying your webapp using Vercel
+
+Now that we have our hello world application we would like to learn how to deploy this app into production. To do that, we are using Vercel for deploying server infrastructure. Let's find out what is Vercel?
+
+Vercel is a cloud platform that specializes in static site hosting and serverless functions deployment. It provides an intuitive and seamless way to deploy web applications, APIs, and serverless functions, with a focus on speed, scalability, and ease of use. Vercel is known for its developer-friendly experience, making it a popular choice for static site hosting, Jamstack applications, and serverless functions deployment. It offers an intuitive user interface, seamless Git integration, automatic scaling, built-in analytics and monitoring, domain and SSL management and a range of powerful features that simplify the deployment and management of modern web applications.
+
+If you would have wanted to deploy this application using AWS services, you would have had to learn about infrastructures such as Cloudfront, API gateways, lambdas, Rout 53, SSL management, S3 buckets and tools to deploy and manage your infrastructure such as CDK. Vercel abstracts all of those complexities awy allowing you to focus on application development rather than managing infrastructure. This brings an incredible speed when you want to get started with building your product idea as well as it eliminating all the costs of maintaining your infrastructure going forward.
```

Comment:
```suggestion
If you wanted to deploy this application using AWS services, you would have had to learn about infrastructure such as CloudFront, API Gateway, Lambda, Route 53, SSL management, S3 buckets and tools to deploy and manage your infrastructure such as CDK. Vercel abstracts all of those complexities away allowing you to focus on application development rather than managing infrastructure. This brings an incredible speed when you want to get started with building your product idea as well as it eliminating all the costs of maintaining your infrastructure going forward.
```